ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/ContractsReader.scala

```diff
@@ -55,6 +63,41 @@ private[dao] sealed abstract class ContractsReader(
           )
       })
 
+  /** Lookup of a contract in the case the contract value is already known (loaded from a cache) */
+  private def lookupActiveContractWithoutArgument(
```

Comment:
Based on the comment shouldn't this be rather called `lookupActiveContractWithValue` or `lookupActiveContractWithKnownValue` instead?

Reply:
Renamed methods for clarity

Reply:
Great, thanks!